addons/account/models/partner.py

```diff
@@ -292,6 +292,11 @@ class AccountFiscalPositionAccount(models.Model):
 class ResPartner(models.Model):
     _name = 'res.partner'
     _inherit = 'res.partner'
+    company_registry_number = fields.Char(string="Company Registry Number", help="The registry number of your company")
```

Comment:
Little blank line between the _inherit and the field definition would be welcome :)

```suggestion
    _inherit = 'res.partner'
    
    company_registry_number = fields.Char(string="Company Registry Number", help="The registry number of your company")
```